README.md

```diff
@@ -1,22 +1,27 @@
 # ParallelChain Mainnet Contract Compiler (pchain_compile) 
 
-`pchain_compile` is a command line tool for reproducibly building Rust code into compact, gas-efficient WebAssembly [ParallelChain Mainnet Smart Contracts](https://github.com/parallelchain-io/parallelchain-protocol/blob/master/Contracts.md).
+`pchain_compile` is a command line tool for reproducibly building [Rust](https://www.rust-lang.org/) code into compact, gas-efficient WebAssembly [ParallelChain Mainnet Smart Contracts](https://github.com/parallelchain-io/parallelchain-protocol/blob/master/Contracts.md).
 
 ## Pre-Requisites
 
-`pchain_compile` builds the source code in a docker environment. To know more about Docker and install it, refer to the [official instructions](https://docs.docker.com/get-docker/).
+`pchain_compile` compiles the ParallelChain smart contract code in Rust into contract binary in WebAssembly. To know more about developing ParallelChain smart contracts, visit [ParallelChain Mainnet Contract SDK](https://crates.io/crates/pchain-sdk).
```

Comment:
Phrasing suggestion, "compiles the ParallelChain smart contract code from Rust into a WebAssembly binary"